backend/trivia_api/views.py

```diff
@@ -61,6 +61,71 @@ def users(user_id):
     return dummy_users
 
 
+@blueprint.route("/signup", methods=["GET", "POST"])
+def signup_post():
+    if request.method == "POST":
+        # return jsonify({"message": "success"}, 200)
+        data = request.get_json("password")
+        newusers = data["username"]
+        #  newusers2 = str(newusers)
+        newpasswords = data["password"]
+        passwords = generate_password_hash(newpasswords, method="sha256")
```

Comment:
```suggestion
        username = data["username"]
        password = data["password"]
```

variable naming, no need to hash if a user isnt going to be made